Sources/KituraWebSocket/WebSocketService.swift

```diff
@@ -49,4 +49,12 @@ public protocol WebSocketService: class {
     /// - Parameter client: The `WebSocketConnection` object that represents the connection over which
     ///                     the client sent the message to this `WebSocketService`
     func received(message: String, from: WebSocketConnection)
+    
+    /// The time in seconds after which, if a connection is has sent no messages, the WebSocket server will send a ping to the connection. If a pong is not received in response, the connection will be closed. If connectionTimeout is nil, no connection cleanup will take place.
+    var connectionTimeout: Int? { get }
+}
+extension WebSocketService {
+    var connectionTimeout: Int? {
```

Comment:
Comment please.

Reply:
Does this need to be marked `public` or is it implicitly public?

Reply:
protocols cannot declare field access levels so that is set when something conforms to the protocol. The comment describing this is on the protocol `connectionTimeout` which is what gets displayed in Jazzy. I will add a comment describing the default implementation though.